lasagne/layers/recurrent.py

```diff
@@ -1,55 +1,98 @@
+"""
+Layers to construct recurrent networks. Recurrent layers can be used similarly
+to feed-forward layers except that the input shape is expected to be
+(batch_size, sequence_length, num_inputs).
+
+The following recurrent layers are implemented:
+
+* :func:`CustomRecurrentLayer()`
+* :func:`RecurrentLayer()`
+* :func:`LSTMLayer()`
+
+Recurrent layers and feed-forward layers can be combined in the same network
+by using a few reshape operations, please refer to the recurrent examples for
+further explanations.
+
+"""
 import theano
 import theano.tensor as T
 from .. import nonlinearities
 from .. import init
-from .. import utils
 
 from .base import Layer
 from .input import InputLayer
 from .dense import DenseLayer
 from . import helper
 
+__all__ = [
+    "CustomRecurrentLayer",
+    "RecurrentLayer",
+    "LSTMLayer"
+]
+
 
 class CustomRecurrentLayer(Layer):
-    '''
+    """
     A layer which implements a recurrent connection.
 
     Expects inputs of shape
         (n_batch, n_time_steps, n_features_1, n_features_2, ...)
-    '''
-    def __init__(self, input_layer, input_to_hidden, hidden_to_hidden,
+
```

Comment:
How come you moved the docstring up here?  These parameters are arguments for the `__init__` function, not the `CustomRecurrentLayer`.  Usually, the docstring of the class includes attributes of the class, while the docstring of the `__init__` method includes arguments to the `__init__` function.  See https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#documenting-classes